pkg/larker/fs/github/github.go

```diff
@@ -0,0 +1,51 @@
+package github
+
+import (
+	"context"
+	"encoding/base64"
+	"errors"
+	"fmt"
+	"github.com/google/go-github/v32/github"
+	"syscall"
+)
+
+var ErrAPI = errors.New("failed to communicate with the GitHub API")
+
+type GitHub struct {
+	client    *github.Client
+	owner     string
+	repo      string
+	reference string
+}
+
+func New(owner, repo, reference, token string) *GitHub {
```

Comment:
`token` is not used.

Reply:
Indeed, fixed in https://github.com/cirruslabs/cirrus-cli/pull/89/commits/728ad2c32bd2817e3bc4a3a19cc0060849aec74a.